Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.cs

```diff
@@ -108,6 +120,17 @@ protected override void OnApplyTemplate()
                 Start();
             }
 
+            if (MinRandomDuration > MaxRandomDuration)
+            {
+                throw new ArgumentException("MinRandomDuration can't be bigger than MaxRandomDuration");
+            }else if (MinRandomDuration == MaxRandomDuration)
```

Comment:
Well, it's kinda getting late in here and I've been working for like 14 hours or so :) Thanks for pointing out. Will prepare docs and update the sample app asap.